### PR TITLE
Margaux/endgame scores, buzzer skip and layout fix

### PR DESCRIPTION
**Pull request by zwierski**
This PR:
- allows the display of all the players' score in the ending screen after a local buzzer game
- adds a skip button to the local buzzer game (the skip button will be added to other game modes later as I had a lot of issues with the layout)
- fixes the issue of the next button appearing over the buzzers in the local buzzer game

### CODE REVIEW

File: app/src/main/java/ch/sdp/vibester/activity/BuzzerScreenActivity.kt [hunk 1]
```diff
@@ -95,6 +96,7 @@ class BuzzerScreenActivity : GameActivity() {
      */
     fun startRoundBuzzer(ctx: Context, gameManager: GameManager) {
         gameIsOn = true
+        toggleBtnVisibility(R.id.skip, true)
         findViewById<LinearLayout>(R.id.answer).visibility=View.INVISIBLE
         val title = gameManager.getCurrentSong().getTrackName()
```
Comment (Tsathogguaa) [inline app/src/main/java/ch/sdp/vibester/activity/BuzzerScreenActivity.kt:101]: Could you rename this to something more indicative, like trackName or something? Title may be confusing to understand without reading the code in the future

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -217,6 +217,7 @@
     <string name="top_tracks">Top Tracks</string>
 
     <string name="PartyRoom_joinedPeople">Joined People:</string>
+    <string name="skip">SKIP</string>
```
Comment (MaximeZmt) [inline app/src/main/res/values/strings.xml:220]: Please use the naming convention (written in wiki)

File: app/src/main/java/ch/sdp/vibester/activity/GameEndingActivity.kt [hunk 3]
```diff
@@ -89,6 +96,24 @@ class GameEndingActivity : AppCompatActivity() {
...
+     * @param text: the text to be put in the view
+     * @param gravity: the desired gravity of the view
+     * @return the created view
+     */
+    private fun createTextView(text: String, gravity: Int, row: TableRow) {
+        val view = TextView(this)
+        view.text = text
+        view.height = 100
+        view.width = 300
+        view.textAlignment=View.TEXT_ALIGNMENT_CENTER
```
Comment (MaximeZmt) [inline app/src/main/java/ch/sdp/vibester/activity/GameEndingActivity.kt:110]:
```suggestion
        view.textAlignment = View.TEXT_ALIGNMENT_CENTER
```

File: app/src/main/java/ch/sdp/vibester/activity/BuzzerScreenActivity.kt [hunk 4]
```diff
@@ -109,7 +111,11 @@ class BuzzerScreenActivity : GameActivity() {
      * Ends the round when no ones answer before the time limit
      */
     fun timeoutAnswer(ctx: Context, chosenSong: Song?=null, gameManager: GameManager) {
+        if (gameManager.playingMediaPlayer()) {
+            gameManager.stopMediaPlayer()
+        }
```
Comment (Tsathogguaa) [inline app/src/main/java/ch/sdp/vibester/activity/BuzzerScreenActivity.kt:116]: Same code block as lines 197-199. Maybe can refactor those into a function?

**Review by MaximeZmt — COMMENTED** [review]
Nice work 👍 I've left minor correction below.

**Review by Tsathogguaa — APPROVED** [review]
Looks good! Two comments made above, which shouldn't take long to fix.

**Review by MaximeZmt — APPROVED** [review]
LGTM 👍

**Pull request merged**